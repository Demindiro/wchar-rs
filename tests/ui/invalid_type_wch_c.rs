use wchar::wch_c;

const INVALID: &[f32] = wch_c!(f32, "oops");

fn main() {}
