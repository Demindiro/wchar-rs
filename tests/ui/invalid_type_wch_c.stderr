error: unexpected wchar type `f32`, expected one of: u16, u32, i16, i32
 --> $DIR/invalid_type_wch_c.rs:3:32
  |
3 | const INVALID: &[f32] = wch_c!(f32, "oops");
  |                                ^^^
